example/android/app/src/main/kotlin/com/mr/flutter/plugin/filepicker/example/MainActivity.kt

```diff
@@ -1,6 +0,0 @@
-package com.mr.flutter.plugin.filepicker.example
```

Comment:
Can you restore this file? It is under the correct path, if we follow the original namespace (com.mr.flutter.plugin.filepicker.example)